website/blog/revocation-ocsp-responder.md

```diff
@@ -0,0 +1,27 @@
+---
+title: OCSP Responder for revocation
+description: Support for free revocation of DID signed documents using an online certificate status responder
+slug: revocation-ocsp-responder
+authors:
+  - name: Wayne Wee
+    title: OpenAttestation Contributor
+    url: https://github.com/waynewee
+    image_url: https://avatars.githubusercontent.com/u/34233605?v=4
+tags: [revocation, did]
+hide_table_of_contents: false
+---
+
+We recently released support for revocation of DID signed documents through and Online Certificate Status Protocol (OCSP) responder which would allow for revocation of documents without the need to deploy a document store. Prior to this release, revocation of documents was only possible through the deployment of a document store which is costly.
```

Comment:
```suggestion
We recently released support for revocation of DID signed documents through an Online Certificate Status Protocol (OCSP) responder which would allow for the revocation of documents without the need to deploy a document store. Prior to this release, revocation of documents was only possible through the deployment of a document store which is costly.
```